tests/e2e/lib/dpa_helpers.go

```diff
@@ -28,12 +28,14 @@ const (
 	CSIDataMover BackupRestoreType = "csi-datamover"
 	RESTIC       BackupRestoreType = "restic"
 	KOPIA        BackupRestoreType = "kopia"
+	VSL          BackupRestoreType = "vsl"
```

Comment:
following  these changes https://github.com/openshift/oadp-operator/commit/68a1c47edd28a9acfd5f93bb76dd08989a6667ac I think is better to rename `vsl` here

Maybe `native-snapshots`?

Reply:
actually its VSL everywhere in the product documentation as well, whenever snapshotLocations are being configured. And its smaller and easy as well, maybe in this https://github.com/openshift/oadp-operator/commit/68a1c47edd28a9acfd5f93bb76dd08989a6667ac we can add  "(VSL)" in brackets?

Reply:
@weshayutin is this going to change in the docs?

Reply:
So VSL is not what we call the backup type -- that would be "native snapshots". More general, the terminology is as follows:
1) "native snapshots" -- Backup via cloud provider snapshots rather than CSI snapshots or filesystem backup.
2) "Volume Snapshotter Plugin" -- plugin that performs the native snapshot backup.
3) "VolumeSnapshotLocation (VSL)" -- the CR that tells Velero how/where the snapshots are saved. Usually this just means configuring the region and credentials for a particular provider.

So VSL is not the functionality as a whole, it's a configuration CR.

Reply:
ack.